pkg/pipeline/ingest/ingest_kafka.go

```diff
@@ -219,15 +218,25 @@ func NewIngestKafka(params config.StageParam) (Ingester, error) {
 		dialer.TLS = tlsConfig
 	}
 
-	kafkaReader := kafkago.NewReader(kafkago.ReaderConfig{
+	readerConfig := kafkago.ReaderConfig{
 		Brokers:        jsonIngestKafka.Brokers,
 		Topic:          jsonIngestKafka.Topic,
 		GroupID:        jsonIngestKafka.GroupId,
 		GroupBalancers: groupBalancers,
 		StartOffset:    startOffset,
 		CommitInterval: time.Duration(commitInterval) * time.Millisecond,
 		Dialer:         dialer,
-	})
+	}
+
+	if jsonIngestKafka.PullBatchLen > 0 {
+		readerConfig.QueueCapacity = jsonIngestKafka.PullBatchLen
```

Comment:
We discussed it last week, but for the record: it would be better to name this params similarly to the corresponding name in the client lib, `QueueCapacity`, also "batch length" isn't exactly the same meaning as "queue capacity".

Reply:
fixed, thanks